int64[] action
---
float64[] position
float64[] orientation
float64 reward
